app/components/Editor.js

```diff
@@ -121,13 +121,14 @@ export default class Editor extends React.Component {
         $R('#editor', 'module.buffer.clear');
       },
       onNoteValueChange: async (note) => {
+        $R('#editor', 'source.setCode', '');
```

Comment:
Should this be handled by EditorKit instead so all editors gain this behavior? Or do we only need it for particular editors?

Reply:
In this case, we need this for this particular editor. But we could definitely integrate this along with the functionality that detects potentially dangerous content into the EditorKit.